### DISCUSSION TITLE
Supported video formats for Slingbox ProHD and 500

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#466] [Q&A]
There used to be some useful webpages with this information on the Sling website but that has now vanished of course.
The reason I am interested is that in the past I have tried to find out why my ProHD would not output video from my UK Virginmedia Tivo V6 box when the HDMI output was sent via an HDMI to Composite converter into the ProHD. Turned out that the Tivo box could not send a  video that the ProHD could handle. The ProHD does handle the HDMI converted output from my Humax Foxsat box just fine.

I was wondering if the Slingbox 500 with its direct HDMI input would handle the HDMI output of my Tivo V6 ? So a list of formats that the 500 CAN handle would be useful.

Alternatively is there anyone out there in the UK that has a Slingbox 500 successfully connected to a Virgin Tivo box ?

  BTW I DO have a table of supported formats for the M1 and 350 if anyone needs it.

Thanks

**Comment by sp222222** [comment]
they wayback web
http://web.archive.org/

http://web.archive.org/web/20221108053712/https://www.slingbox.com/

actually this 
http://web.archive.org/web/20210307075724/https://www.slingbox.com/help/KB/kb-2000537
